src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You should add a fallback font in case the primary font doesn't load. For example, you could change this to `font-family: Roboto, Arial, sans-serif, sans-serif;`.